### DISCUSSION TITLE
Setup Mobile & Computer

### DISCUSSION THREAD

**Discussion by kennygokh** [chhoumann/quickadd#610] [Q&A]
Hi, is it possible to sync the QuickAdd setup like capture, macro etc configuration? I have setup in my macbook pro and I realised that I need to re-setup in iPhone. Did I miss some steps? How to sync the settings on both devices so that I can setup only once. Please advise.

**Comment by chhoumann** [comment]
Coming back to this one very late - sorry about that.

QuickAdd's whole configuration (captures, macros, all your choices) lives in one file inside your vault: `.obsidian/plugins/quickadd/data.json`. So you don't need to redo the setup on iPhone - you just need your sync solution to carry the `.obsidian` folder across:

- **Obsidian Sync**: in the Sync settings on both devices, turn on syncing for installed community plugins and their settings, and make sure both devices use the same config folder name.
- **iCloud or any whole-vault file sync**: works automatically, since `.obsidian` is part of the vault.

One caveat: if your macros reference user scripts, keep those `.js` files inside the vault so they sync too. Scripts that rely on desktop-only Node APIs won't run on iOS, but everything else does.

If you'd rather move things explicitly instead of syncing, QuickAdd can also export choices (including their scripts) as a package you import on the other device: https://quickadd.obsidian.guide/docs/Choices/Packages
